common/v2/utils/environment.ts

```diff
@@ -2,4 +2,5 @@ export const IS_DEV: boolean = process.env.NODE_ENV === 'development';
 
 export const IS_PROD: boolean = process.env.NODE_ENV === 'production';
 
-export const HAS_WEB3_PROVIDER: boolean = window && 'web3' in window;
+export const HAS_WEB3_PROVIDER = (): boolean =>
+  window && ('web3' in window || 'ethereum' in window);
```

Comment:
Since this is a function now, maybe we should change the syntax to camelCase or any other convention we decide on to not be surprised by `HAS_WEB3_PROVIDER()`. As you see fit